site/components/navigator/index.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import React from 'react';
+import classnames from 'classnames';
+import styles from './style.css';
+
+type Props = {
+  currentIndex: number,
+  maxIndex: number,
+  onChange: number => void,
+};
+
+function mapOverRange(count, mapper) {
+  const array = [];
+
+  for (let i = 0; i < count; i++) {
+    array.push(mapper(i));
+  }
+
+  return array;
+}
+
+const Navigator = ({ currentIndex, maxIndex, onChange }: Props) => {
```

Comment:
Could define a renderArrow function to cut back on some of the duplication. Something like:

```jsx
const renderArrow = (direction = 'left') => (
  <Arrow
    direction={direction}
    onClick={onChange}
    currentIndex={currentIndex}
    lastIndex={maxIndex}
  />
);
```

```jsx
{renderArrow('right')}
```